### ISSUE TITLE
multi-label text classifiers

### ISSUE THREAD

**Issue by iLOSTA** [iLOSTA/The-Project#3]

**Comment by iLOSTA** [comment]
I have chosen and successfully implemented 6 different classification algorithms: SVC, LogisiticRegression, MLkNN, MultinominalNB, NN and Random Forests. some algorithms performed very poorly like Naive Bayes and Logistic Regression, but the other algorithms are giving promising results:
MLkNN : 0.88 accuracy
SVC : 0.88
NN: 0.98 Accuracy!! (currently investigating this)
Also, tried each of these classifiers with bi-gram and regular 1 word tf-idf, there seems to be no difference in the accuracy between the two however.

**Comment by iLOSTA** [comment]
as a final note on this ticket I'd like to say that:

-  It turned out I didn't try Random Forests as a multi-label classifier. Instead I tried it as a multi-class classifier on the 'MAKE' class only and never continued working on it simply because it performed very poorly there (73% accuracy). I thought I'd save myself the trouble of transforming it into a multi-label classifier. 

- Five classifiers were implemented and compared: Naive Bayes (62%), Logistic Regression (60%), Support Vector Machine SVC (90%), Multi-label k-NN (94%) and NN (99.7%).

- From now on, I will be focusing on NN only.
